src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -197,45 +257,79 @@ public Integer updateAppsUsageAssociationData(
      * @param studyId - study id
      * @return true if notifications are enabled for a given study
      */
+    @Deprecated( since = "apps v2" )
     @GET( BASE + STUDY_ID_PATH + NOTIFICATIONS )
     Boolean isNotificationsEnabled(
             @Path( STUDY_ID ) UUID studyId
     );
 
+    /**
+     * Apps V2: Verify that daily push notifications are enabled for participant devices associated with a study
+     *
+     * @param organizationId - Id of organization to which study belongs
+     * @param studyId        - study id
+     * @return true if notifications are enabled for a given study
+     */
+    @GET( BASE + ORGANIZATION_ID_PATH + STUDY_ID_PATH + NOTIFICATIONS )
+    Boolean isNotificationsEnabledV2(
+            @Path( ORGANIZATION_ID ) UUID organizationId,
+            @Path( STUDY_ID ) UUID studyId
+    );
+
     /**
      * Query the enrollment status of a participant
      *
      * @param studyId       - studyId
      * @param participantId - participantId
      * @return one of { ENROLLED, NOT_ENROLLED, UNKNOWN }
      */
+    @Deprecated( since = "apps v2" )
     @GET( BASE + STUDY_ID_PATH + PARTICIPANT_ID_PATH + ENROLLMENT_STATUS )
     ParticipationStatus getParticipationStatus(
             @Path( STUDY_ID ) UUID studyId,
             @Path( PARTICIPANT_ID ) String participantId
     );
 
+    /**
+     * Apps v2: Query the enrollment status of a participant
+     *
+     * @param organizationId - Id of the organization to which study belongs
+     * @param studyId        - studyId
+     * @param participantId  - participantId
+     * @return one of { ENROLLED, NOT_ENROLLED, UNKNOWN }
+     */
+    @GET( BASE + ORGANIZATION_ID_PATH + STUDY_ID_PATH + PARTICIPANT_ID_PATH + ENROLLMENT_STATUS )
+    ParticipationStatus getParticipationStatusV2(
+            @Path( ORGANIZATION_ID ) UUID organizationId,
+            @Path( STUDY_ID ) UUID studyId,
+            @Path( PARTICIPANT_ID ) String participantId
+    );
+
     /**
      * Retrieve questionnaire matching given entity key id
      *
+     * @param organizationId    - Id of the organization to which study belongs
      * @param studyId           - studyId
      * @param questionnaireEKID - questionnaire entity key id
      */
-    @GET( BASE + STUDY_ID_PATH + QUESTIONNAIRE + ENTITY_KEY_ID_PATH )
+    @GET( BASE + ORGANIZATION_ID_PATH + STUDY_ID_PATH + QUESTIONNAIRE + ENTITY_KEY_ID_PATH )
```

Comment:
This looks like a breaking change.

Reply:
This doesn't need to be backward compatible since it's only used by the web app at the moment.